### PR TITLE
administration.md / 5.4.3 Batch upload via phpMyAdmin

### PR DESCRIPTION
**Pull request by c1972**
Added 4.5.1 Batch upload FAQs to Database with CSV files.

### CODE REVIEW

**Comment by thorsten** [comment]
usage.md is for the public frontend, it should be in administration.md

**Review by thorsten — CHANGES_REQUESTED** [review]
Please add this to administration.md

**Comment by c1972** [comment]
I now have it in administration.md as point 5.4.3 Batch upload via phpMyAdmin. Did it arrive?
(Sorry, it's taking me a while to find my way around here.)

**Comment by c1972** [comment]
I've now submitted it a third time...

- first switched my phpMyAdmin to English
- then took 4 screenshots
- edited them with Photoshop
- saved them as PNG files
- compressed them
- uploaded them to my webspace
- included the 4 screenshots in the docs
- fixed a few more errors

**Comment by c1972** [comment]
I have also adjusted my forum post accordingly.
https://forum.phpmyfaq.de/viewtopic.php?f=2&t=43445

File: docs/administration.md [hunk 1]
```diff
@@ -253,6 +253,65 @@ You can export your contents of your whole FAQ or just some selected categories
...
+> 
+> This guide is intentionally extensive, but the individual steps can be completed quite quickly once you understand them. It is important that you don't make any mistakes when uploading to your DB, so take your time the first time you try it.
+
+### Export the necessary tables
+
+Open phpMyAdmin. First check whether the correct database is displayed. You can switch to a different database at the top left. At the top left you can see the name of your database and below that the names of the individual tables. From this list, select the following 4 tables (one after the other) and click on them once to select them:
+* faqdata
+* faqdata_tags
+* faqcategoryrelations
+* faqdata_user.csv
```
Comment (thorsten) [inline docs/administration.md:273]: faqdata_groups is missing

Comment (thorsten) [inline docs/administration.md:273]: I think it's just "faqdata_user"

File: docs/administration.md [hunk 2]
```diff
@@ -253,6 +253,65 @@ You can export your contents of your whole FAQ or just some selected categories
...
+
+![Screenshot of choosing DB and tables in phpmyadmin.](https://pokerfit.de/forum/phpmyadmin-sc-1.png)
```
Comment (thorsten) [inline docs/administration.md:275]: The images should be moved to this repo

File: docs/administration.md [hunk 3]
```diff
@@ -253,6 +253,65 @@ You can export your contents of your whole FAQ or just some selected categories
...
+
+As soon as you have selected one of the 4 tables, the contents will be displayed on the right. Now click on the **Export** menu item at the top.
```
Comment (thorsten) [inline docs/administration.md:277]: 5 tables as mentioned above

File: docs/administration.md [hunk 4]
```diff
@@ -253,6 +253,65 @@ You can export your contents of your whole FAQ or just some selected categories
...
+
+![Screenshot of nav bar in phpmyadmin.](https://pokerfit.de/forum/phpmyadmin-sc-2.png)
+
+Select **CSV** as the format. Make sure that **Dump all rows** is selected. Then click **Export** at the bottom.
+
+![Screenshot of export dialog in phpmyadmin.](https://pokerfit.de/forum/phpmyadmin-sc-3.png)
+
+Save all 4 CSV-Files in a separate folder on your computer. Then you can close phpMyAdmin for the moment.
```
Comment (thorsten) [inline docs/administration.md:285]: 5 tables

File: docs/administration.md [hunk 5]
```diff
@@ -253,6 +253,65 @@ You can export your contents of your whole FAQ or just some selected categories
...
+
+Within the Tags section of phpMyFAQ you can hover your mose over the red trash can and your browser will now show you a link at the bottom left that ends with our tag number. You should note down the tags and their numbers in a text file to have them at Hand later. With this knowledge we can now enter the desired tags in the table for our new posts. Check again and you can close the table.
+
+We continue with the third table, called *faqcategoryrelations*, and this is now about the categories. The number in the first column category_id identifies the category in which the new post should be sorted. You can assign the number to the category in exactly the same way as you did with the tags. So make a quick note in your text file! You can now enter the numbers so that the posts end up in the right category. On the right at record_id we have the unique ID that identifies the post. Once you have entered everything, save and close.
+
+The fourth file called *faqdata_user.csv* is easy. Just put in the *unique id* of your new entries at the left and *-1* at the right. Save and close.
```
Comment (thorsten) [inline docs/administration.md:303]: You need the faqdata_groups table as well, it's also just the ID of the entries and -1

File: docs/usage.md [hunk 6]
```diff
@@ -58,6 +58,46 @@ New FAQ entries won't be displayed by default and have to be activated by an adm
 
 If a user is logged in, the name and e-mail address are filled automatically.
 
+## 4.5.1 Batch upload FAQs to Database with CSV files
```
Comment (thorsten) [inline docs/usage.md:61]: This change should be reverted as it's moved to administration.md

**Review by thorsten — CHANGES_REQUESTED** [review]

File: docs/administration.md (hunk 2 above)
Comment (c1972) [reply docs/administration.md:275]: How can I move the images to this repo? I searched a while, but didn't find it :(

**Comment by c1972** [comment]
I think I miss understood your wish to revert a change. You want me to delete the whole point in usage.md right?

**Review by thorsten — APPROVED** [review]

**Pull request merged**